doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -54,6 +54,11 @@ Matter
 
 * Added ``EXPERIMENTAL`` select in Kconfig that informs that Matter support is experimental.
 
+Thread
+------
+
+* Thread 1.2 enabling method has changed, refer to :ref:`thread_ug_thread_specification_options` and :ref:`thread_ug_feature_sets` for more information.
```

Comment:
```suggestion
* Changed how Thread 1.2 is enabled, refer to :ref:`thread_ug_thread_specification_options` and :ref:`thread_ug_feature_sets` for more information.
```
To have it in the same tense as the other entries.

Reply:
fixed